ProjectManagmentSystem/src/Modules/AuthenticationModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,147 @@
-import React from 'react'
 
+import logo from '../../../../Modules/../assets/images/PMS 3.svg'
+import { TextField } from '@mui/material';
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 export default function ResetPass() {
+  let  navigate = useNavigate()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit =  async (data) => {
+    console.log(data)
+    try {
+      let response = await axios.post("https://upskilling-egypt.com:3003/api/v1/Users/Reset",data)
```

Comment:
Having the URL hardcoded like this is not a good practice, as this means that if we have the URL in 100 component and the URL has changed that we need to manually update all URLS.

An alternative approach to use axios.create to setup an axios instance that have the base URL and any other configuration, and when using this instance, we will pass only the relative path to the base URL.

We can also store the URL's in a constants file like this one:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/APIs.ts